src/config/bookconfig.rs

```diff
@@ -18,6 +18,9 @@ pub struct BookConfig {
     indent_spaces: i32,
 
     html_config: Option<HtmlConfig>,
+
+    use_ace: bool,
```

Comment:
Maybe we could have these abstracted into another struct along the lines of `Option<PlaypenEditor>`  (or `Option<PlaypenEditorConf>` )